src/module/item/sheet.js

```diff
@@ -256,12 +256,19 @@ export class ItemSheetSFRPG extends ItemSheet {
             )
         );
         } else if (item.type === "spell") {
+            const desc = (Object.entries(itemData.descriptors)).filter(e => e[1] === true).map(e => ({
```

Comment:
The open parentheses after the `=>` is unnecessary here. Just need the curly bracket.

Reply:
I noticed this when writing this code. It throws an error when those brackets aren't there. You'll see that the weapon property code on line 252 also has those brackets (that's where I copied the code from in the first place)

Reply:
Ok, I see what its doing here. It looks funky, but its working they way it should, so never mind ;)